### DISCUSSION TITLE
Multiple Routing instances,Unicast Service Discovery Offer and Request,defined UDP port from Client

### DISCUSSION THREAD

**Discussion by athul-am** [COVESA/vsomeip#813] [Q&A]
Hi all,
I have a scenario where i am using a board that has 4 ethernet ports. I would like to create 4 Applications or Instances where each one is listening on each port is this possible? if so how?

The other 2 situations are I need to send the Service Discovery using Unicast and the the Subscription port of my client i need to keep that from the someip config .Please let me know if anyone has done this . It would be a great help!!!

Thanks,
Athul

**Comment by duartenfonseca** [comment]
each Ethernet port should have an address assigned to it, and each vsomeip application should use one of them. you also have to configure different settings for the configuration variable "network".

maybe you can clarify the other 2 situations.
